Formula/brev.rb

```diff
@@ -0,0 +1,17 @@
+class Brev < Formula
+  desc "CLI tool for managing workspaces provided by brev.dev"
+  homepage "https://docs.brev.dev"
+  url "https://github.com/brevdev/brev-cli/archive/refs/tags/v0.6.17.tar.gz"
+  sha256 "cf69be0558cd87ca5838372aeca6b85fe937f1bd8eb44c03a88355ca088aa9ea"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}"
+    system "go", "build", *std_go_args(output: bin/"brev", ldflags: ldflags)
+  end
+
+  test do
+    system "#{bin}/brev", "healthcheck"
```

Comment:
Does this return a non-zero exit code if there's an issue)

Reply:
yes
```
λ brev healthcheck
Healthy!
λ echo $?
0
λ brev healthcheck
Not Healthy!
Error: [error] github.com/brevdev/brev-cli/pkg/cmd/healthcheck/healthcheck.go:34 
	: [error] github.com/brevdev/brev-cli/pkg/cmd/healthcheck/healthcheck.go:47 
	: [error] github.com/brevdev/brev-cli/pkg/store/healthcheck.go:10 
	: Get "https://ade5dtvtaa.execute-api.us-east-1.amazonaws.com/api/health?utm_source=cli": dial tcp: lookup ade5dtvtaa.execute-api.us-east-1.amazonaws.com: Temporary failure in name resolution
Usage:
  brev healthcheck [flags]

Examples:
brev healthcheck

Flags:
  -h, --help   help for healthcheck

Global Flags:
      --version   Print version output

λ echo $?
1
```